sky/check.py

```diff
@@ -267,16 +268,63 @@ def get_cloud_credential_file_mounts(
     return file_mounts
 
 
-def _format_enabled_cloud(cloud_name: str) -> str:
+def _print_checked_cloud(
+    echo: Callable, verbose: bool, cloud_tuple: Tuple[str,
+                                                      Union[sky_clouds.Cloud,
+                                                            ModuleType]],
+    cloud_capabilities: List[Tuple[sky_cloud.CloudCapability, bool,
+                                   Optional[str]]]
+) -> None:
+    cloud_repr, cloud = cloud_tuple
+    # Print the capabilities for the cloud.
+    # consider cloud enabled if any capability is enabled.
+    enabled_capabilities: List[sky_cloud.CloudCapability] = []
+    hints_to_capabilities: Dict[str, List[sky_cloud.CloudCapability]] = {}
+    reasons_to_capabilities: Dict[str, List[sky_cloud.CloudCapability]] = {}
+    for capability, ok, reason in cloud_capabilities:
+        if ok:
+            enabled_capabilities.append(capability)
+            if reason is not None:
+                hints_to_capabilities.setdefault(reason, []).append(capability)
+        elif reason is not None:
+            reasons_to_capabilities.setdefault(reason, []).append(capability)
+    status_msg = 'enabled' if enabled_capabilities else 'disabled'
+    styles = {
+        'fg': 'green',
+        'bold': False
+    } if enabled_capabilities else {
+        'dim': True
+    }
```

Comment:
Let's collapse this or just make it an `if`: https://google.github.io/styleguide/pyguide.html#2114-decision